### PR TITLE
feat: Adds implementation of forecast function

### LINKED ISSUE #11: Implement and write tests for get_pollution_forecast
- [x] Write function
- [x] Exception handle
- [ ] Unit tests

### PR DESCRIPTION
**Pull request by christopheralex**
- [x] Adds implementation to forecast function.
- [x] Test cases for forecast function.
- [x] Exception handling for forecast function.
- [x] Installes corresponding dependencies.

### CODE REVIEW

File: R/get_pollution_forecast.R [hunk 1]
```diff
@@ -15,4 +21,90 @@
...
+        stop("An error occured in plotting")
+      }
+      }
+    else{
+      stop("Insufficient data to forecast/plot.")
+    }
+  }
+  else{
+    print('HERE????')
+    print(api_error)
```
Comment (mel-liow) [inline R/get_pollution_forecast.R]: When running the test, i found that the mock api error was coming to this line here.
This was the output of the `print(api_error)` which told me we needed to add data in `forecast-668776.json`
Screenshot 2022-01-25 at 08 02 00

Comment (christopheralex) [reply R/get_pollution_forecast.R]: Thanks . Works now .

File: R/get_pollution_forecast.R [hunk 2]
```diff
@@ -15,4 +21,88 @@
...
+        },
+        error = function(er) {
+          er
+          }
+        )
+      if(!inherits(chart_error,"error")){
+        return(chart)
+      }
+      else{
+        print(chart)
```
Comment (mel-liow) [inline R/get_pollution_forecast.R]: I think we can remove this print statement

Comment (christopheralex) [reply R/get_pollution_forecast.R]: Done :)

**Review by mel-liow — APPROVED** [review]

**Pull request merged**